P2 - Plotting.ipynb

```diff
@@ -0,0 +1,3928 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.    import gravann.polyhedral
</pre><p>This line fails for me with </p><p> </p><pre class="ql-syntax" spellcheck="false">---------------------------------------------------------------------------
ModuleNotFoundError                       Traceback (most recent call last)
&lt;ipython-input-1-2faa20940180&gt; in &lt;module&gt;
      3 
      4 # Import our module containing helper functions
----&gt; 5 import gravann.polyhedral
      6 
      7 # plotting stuff

ModuleNotFoundError: No module named 'gravann.polyhedral'
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/gomezzz/geodesyNets/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>